content/pages/docs/getting-started.md

```diff
@@ -153,6 +153,10 @@ This should bring up a console like this:
 
 <img class="screenshot" alt="Web console connection view" src="/images/webui-connection.png">
 
+## Installing a Browser Extension
+
+You can install a browser extension such as [IPFS Gateway Redirect](https://addons.mozilla.org/firefox/addon/ipfs-gateway-redirect/) for Firefox or [IPFS Station](https://chrome.google.com/webstore/detail/ipfs-station/kckhgoigikkadogfdiojcblegfhdnjei) for Chrome. These will automatically redirect IPFS gateway requests to your local daemon so that you are not relying on, or trusting, remote gateways.
```

Comment:
A lot changed since then. Name. URLs. 
Official browser extension is now named `ipfs-companion` and URLs are:

- Firefox: https://addons.mozilla.org/en-US/firefox/addon/ipfs-companion/
- Chrome: https://chrome.google.com/webstore/detail/ipfs-companion/nibjojkomfdiaoajekhjakgkdhaomnch


Both stores provide own badges too. 
Companion's README displays them like this:

| Firefox                                                                                                                                                    | Chrome / Chromium                                                                                                                                                                        |
| -------------                                                                                                                                              | -------------                                                                                                                                                                            |
| [![Get the add-on](https://blog.mozilla.org/addons/files/2015/11/AMO-button_1.png)](https://addons.mozilla.org/en-US/firefox/addon/ipfs-companion/) | [![](https://developer.chrome.com/webstore/images/ChromeWebStore_BadgeWBorder_v2_206x58.png)](https://chrome.google.com/webstore/detail/ipfs-companion/nibjojkomfdiaoajekhjakgkdhaomnch) |


```
| Firefox                                                                                                                                                    | Chrome / Chromium                                                                                                                                                                        |
| -------------                                                                                                                                              | -------------                                                                                                                                                                            |
| [![Get the add-on](https://blog.mozilla.org/addons/files/2015/11/AMO-button_1.png)](https://addons.mozilla.org/en-US/firefox/addon/ipfs-companion/) | [![](https://developer.chrome.com/webstore/images/ChromeWebStore_BadgeWBorder_v2_206x58.png)](https://chrome.google.com/webstore/detail/ipfs-companion/nibjojkomfdiaoajekhjakgkdhaomnch) |
```